manifests/01-registry-credentials-request-alibaba.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-image-registry
```

Comment:
I suspect you cannot have two credentials requests with the name `openshift-image-registry` and this one should be named like `openshift-image-registry-alibaba`.

Reply:
the name has changed.